packages/next-tinacms-cloudinary/package.json

```diff
@@ -36,5 +36,11 @@
   "repository": {
     "url": "https://github.com/tinacms/tina-graphql-gateway.git",
     "directory": "packages/next-tinacms-cloudinary"
+  },
+  "peerDependencies": {
+    "styled-components": "*"
+  },
+  "optionalDependencies": {
+    "tina-graphql-gateway": "workspace:packages/tina-graphql-gateway"
```

Comment:
I don't think this is optional since we're directly importing it [here](https://github.com/tinacms/tina-graphql-gateway/pull/279/files#diff-4d0e6c70071895ec7caa69e6bac89b0532ebc62274d4b432a254ddf27fc49f6fR15)

Reply:
👍  Good point I will update that.